chainerrl/misc/collections.py

```diff
@@ -0,0 +1,129 @@
+import itertools
```

Comment:
Add future imports (at least `range` is affected)

Reply:
Fixed.